New logo

thibaultduponchelle opened the pull request:
# Description
Replaced old low quality smoking onion per new svg camel logo.

## Preview
![coresmoke-camel](https://github.com/user-attachments/assets/c724648a-d192-46f9-83ad-4f5c2e56f5d5)


## Smoke?
I did not applied the animation since it was not responsive and broken on smartphone.
![smoke-broken-animation](https://github.com/user-attachments/assets/66198232-f412-43a0-a869-4a89733ac84a)

thibaultduponchelle committed:
New logo

smonff commented:
![smokedb](https://github.com/user-attachments/assets/cfefaa01-e82b-4960-ab91-efe6aa6cbaa8)


^^ @thibaultduponchelle

thibaultduponchelle committed:
Improved orange logo

Co-authored-by: Sébastien Feugère <sebastien@feugere.net>

thibaultduponchelle commented:
Thank you a lot @smonff

Updated the pull request with new logo and added you as `Co-authored-by` 

## Preview 
The result looks really gorgeous.

![smokedb-orange-logo](https://github.com/user-attachments/assets/229e0914-4539-4441-8ab6-16c7636f1873)
